src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseStatistics.cpp

```diff
@@ -205,27 +199,30 @@ void CompositionalMultiphaseStatistics::computeRegionStatistics( real64 const ti
   for( integer i = 0; i < regionNames.size(); ++i )
   {
     ElementRegionBase & region = elemManager.getRegion( regionNames[i] );
-    RegionStatistics & regionStatistics = region.getReference< RegionStatistics >( viewKeyStruct::regionStatisticsString() );
 
-    regionStatistics.averagePressure = 0.0;
-    regionStatistics.maxPressure = 0.0;
-    regionStatistics.minPressure = LvArray::NumericLimits< real64 >::max;
+    RegionStatistics & regionStatistics = region.getGroup< RegionStatistics >( viewKeyStruct::regionStatisticsString() );
```

Comment:
To prevent long lines
```suggestion
    RegionStatistics & regionStatistics = 
      region.getGroup< RegionStatistics >( viewKeyStruct::regionStatisticsString() );
```